modular_zubbers/code/modules/power/powerator.dm

```diff
@@ -0,0 +1,14 @@
+/obj/item/circuitboard/machine/powerator/interdyne
+	name = "Ancient Powerator"
+	greyscale_colors = CIRCUIT_COLOR_SUPPLY
+	build_path = /obj/machinery/powerator/interdyne
+
+/obj/machinery/powerator/interdyne
+	name = "ancient powerator"
+	desc = "Beyond the ridiculous name, it is the standard for transporting and selling energy to power networks that require additional sources! It appears to be an earlier variant before environmental regulation reduced its efficiency."
+	circuit = /obj/item/circuitboard/machine/powerator/interdyne
+
+	/// how much the current_power is divided by to determine the profit
+	divide_ratio = 0.00001
```

Comment:
This comes inherited from parent.

```suggestion
```

Reply:
> I **would like** to keep this value despite it being OP, although with the context of a turbine only station it will be fine.

Honestly a fair point, they are running turbine only so it makes sense